kyaml/fn/framework/framework.go

```diff
@@ -116,8 +116,21 @@ func Execute(p ResourceListProcessor, rlSource *kio.ByteReadWriter) error {
 	}
 	rl.FunctionConfig = rlSource.FunctionConfig
 
+	// We store the original
+	nodeAnnos, err := kio.PreprocessResourcesForInternalAnnotationMigration(rl.Items)
```

Comment:
Can we try adding a `defer` statement after this to clear the annotation? Want to see if that will fix the tests in api and cmd/config

Reply:
Discussed offline, apparently this doesn't work. Will take a closer look.